jina/docker/hubio.py

```diff
@@ -31,7 +31,7 @@
 
 _allowed = {'name', 'description', 'author', 'url',
             'documentation', 'version', 'vendor', 'license', 'avatar',
-            'jina_version', 'platform', 'update', 'keywords'}
+            'platform', 'update', 'keywords'}
```

Comment:
If it's not there will it be *allowed* to be added?

Reply:
Well, we did accept to remove this right? I just want to remove the warning because it seems missleading

Reply:
Ok. Then maybe `_allowed` is the wrong name for this variable. It should be `_expected` or something like that

Reply:
I think we do not need all of them

Reply:
But then it's not about being `allowed`, it's about being `required`, perhaps

Reply:
I do not know what's the best logic or if there are required fields @deepankarm ?

Reply:
We have a list of `required` & a list of `_allowed`. `ValueError` is raised if something is missing from `required` & a warning is given if something is missing from `_allowed`. 
https://github.com/jina-ai/jina/blob/master/jina/docker/hubio.py#L590